frontend/src/mlmd/MlmdUtils.ts

```diff
@@ -377,6 +377,13 @@ export function filterLinkedArtifactsByType(
   return artifacts.filter(x => artifactTypeIds.includes(x.artifact.getTypeId()));
 }
 
+export function getExecutionName(execution: Execution): string | undefined {
+  return execution
+    .getCustomPropertiesMap()
+    .get('display_name')
+    ?.getStringValue();
+}
```

Comment:
Moved this function to the `MlmdUtils.ts` file as it is now used in multiple location and better fits alongside the existing `getArtifactName` function.

Reply:
Sounds good - done here: https://github.com/kubeflow/pipelines/pull/8026/commits/b280627cc2364ddfdd90af1ae1b980039454927d